addons/website_slides/models/slide_slide.py

```diff
@@ -371,6 +368,11 @@ def _generate_signed_token(self, partner_id):
             self.write({'access_token': self._default_access_token()})
         return self._sign_token(partner_id)
 
+    @api.depends('channel_id.can_publish')
```

Comment:
HumanLinter: to be moved with other compute methods

Reply:
Done.